data/api-impl/src/commonMain/kotlin/io/github/droidkaigi/confsched2019/data/api/ApiEndpoint.kt

```diff
@@ -0,0 +1,3 @@
+package io.github.droidkaigi.confsched2019.data.api
+
+internal expect fun apiEndpoint(): String
```

Comment:
Thanks for https://github.com/DroidKaigi/conference-app-2019/pull/246#issuecomment-452247536